### ISSUE TITLE
What if the optimization hasn't done yet with the 1.1000 loss function due to a low iteration. Fine tune the model or?

### ISSUE THREAD

**Issue by BestSonny** [BVLC/caffe#1292]
After several hours of trainning,my net just work at accuracy 70% with 200,000 iteration.But the loss function hasn't reach near the zero(just 1.1 when it done).So I consider I just set a not enough iteration.
With this already tained model.
How can I continue to iterate?Should I use fine-tune or something else?
